packages/components/list/src/List.tsx

```diff
@@ -0,0 +1,49 @@
+import { cx, css } from 'emotion';
+import React from 'react';
+
+type ListType = 'ul' | 'ol';
+
+export type ListProps = {
+  // Type of list
+  element?: ListType;
+  // Additional className
+  className?: string;
+  // The children of the List
+  children: React.ReactNode;
+  // Class used for testing purposes
+  testId?: string;
+};
+
+function _List(
+  {
+    element: Tag = 'ul',
+    className,
+    children,
+    testId = 'cf-ui-list',
+    ...otherProps
+  }: ListProps,
+  ref,
+) {
+  return (
+    <Tag
+      data-test-id={testId}
+      className={cx(
+        css({
+          margin: 0,
+          listStyleType: Tag === 'ul' ? 'disc' : 'decimal',
+        }),
+        className,
+      )}
+      ref={ref}
+      {...otherProps}
+    >
+      {children}
+    </Tag>
+  );
+}
```

Comment:
I think we should use our polymorphic components here, maybe just a `Box` that can be either `ul` or `ol`? That would also give us the shared props like `testId` and `className` 🙂